### DISCUSSION TITLE
Is there a way to disable require-jsdoc checks?

### DISCUSSION THREAD

**Discussion by joket1999** [eslint/eslint#16860] [Q&A] [answered]
### Environment

Node version: v14.21.1
npm version: v6.14.17
Local ESLint version: v8.33.0
Global ESLint version: v8.33.0
Operating System: MacOS 10.15.7

### What parser are you using?

@babel/eslint-parser

### What did you do?

<details>
<summary>Configuration</summary>

```
{
"require-jsdoc": ["warn", {
	"require": {
		"FunctionDeclaration": false,
		"MethodDefinition": false
		"ClassDeclaration": false,
		"ArrowFunctionExpression": false
		"FunctionExpression": false
	}
}]
}
```
</details>

```js
function *generator() {
	yield '44';
	yield '55';
}
```

### What did you expect to happen?

Is there a way to disable require-jsdoc checks on generator function?
<img width="770" alt="图片" src="https://user-images.githubusercontent.com/6611880/216805652-5c822d3f-ca7d-454d-9e20-c5b60720d4b4.png">

### What actually happened?

eslint: warning
require-jsdoc - Missing JSDoc comment.

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by Rec0iL99** [comment, marked as answer]
Hi @joket1999,

You can disable the `require-jsdoc` for the `generator` function by using the `eslint-disable-next-line` comment like below:

```js
// eslint-disable-next-line require-jsdoc
function *generator() {
	yield '44';
	yield '55';
}
``` 

This should only disable the `require-jsdoc` rule for the `generator` function.

Hope this helps! Thanks for using ESLint!

**Reply by joket1999** [reply]
Thks. I think that‘s a only way.
